edgelet/iotedge/src/main.rs

```diff
@@ -314,7 +346,7 @@ fn run() -> Result<(), Error> {
                 .unwrap_or_default();
             let since = args
                 .value_of("since")
-                .and_then(|a| a.parse::<i32>().ok())
+                .and_then(|s| parse_since(s).try_into().ok())
```

Comment:
This will silently swallow an invalid value and use the default. We should print an error instead.

Same for `tail` above, and for `tail` and `since` in the new subcommand below.